.coveragerc

```diff
@@ -0,0 +1,4 @@
+[report]
+exclude_lines =
+    pragma: no cover
+    @abstract
```

Comment:
Why is it needed?

Reply:
To exclude abstract classes from coverage (found [here](https://stackoverflow.com/a/41284681)) since they can't be tested. In this case the "request" function from BaseClient.

Reply:
Makes sense!